hooks/connections.js

```diff
@@ -14,21 +14,21 @@ module.exports = function Connections(cb) {
 
     async.mapLimit(connections, 5, function(connectionName, fn) {
         var connectionInfo = self._config.connections[connectionName];
+        // TODO is this being used?
         connectionInfo.name = connectionName;
 
-        // verify adapter info, be forgiving with spelling
-        var adapter = connectionInfo.adapter || connectionInfo.adaptor || false;
+        // verify adapter info
+        var adapter = connectionInfo.adapter || false;
         if (!adapter) return fn('Missing adapter for connection: ' + connectionName);
 
-
         // verify the adapter implements a `registerConnection` method
         if (!adapter.registerConnection || !_.isFunction(adapter.registerConnection) || adapter.registerConnection.length !== 2) {
             return fn('Invalid adapter api');
         }
 
-        adapter.registerConnection(connectionInfo, function(err, connection) {
+        adapter.registerConnection(connectionInfo.config, function(err, connection) {
```

Comment:
That was my first thought, but what if 'adapter' happens to be a necessary attribute for whatever reason....I know the odds of that are slim.